src/Tizen.NUI/src/public/Extents.cs

```diff
@@ -145,8 +155,18 @@ public ushort Start
         /// <summary>
         /// The End extend.
         /// </summary>
+        /// <remarks>
+        /// The setter is deprecated in API8 and will be removed in API10. Please use new Extents(...) constructor.
```

Comment:
Please see above.

Reply:
thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118